packages/font-icons-mdl2/src/index.ts

```diff
@@ -27,15 +27,15 @@ declare global {
   // eslint-disable-next-line @typescript-eslint/naming-convention
   interface Window {
     FabricConfig?: {
-      fontBaseUrl?: string;
+      iconBaseUrl?: string;
     };
   }
 }
 
 const win = getWindow();
 
 export function initializeIcons(
-  baseUrl: string = win?.FabricConfig?.fontBaseUrl ?? DEFAULT_BASE_URL,
+  baseUrl: string = win?.FabricConfig?.iconBaseUrl ?? DEFAULT_BASE_URL,
```

Comment:
Can't make breaking changes like this. Fall back to the deprecated one if the new one isn't there.